src/audio/copier/copier_generic.c

```diff
@@ -280,11 +280,46 @@ pcm_converter_func get_converter_func(const struct ipc4_audio_format *in_fmt,
 				      enum ipc4_direction_type dir)
 {
 	enum sof_ipc_frame in, in_valid, out, out_valid;
+	enum ipc4_sample_type in_type;
+	enum ipc4_sample_type out_type;
+
+	in_type = in_fmt->s_type;
+	out_type = out_fmt->s_type;
+
+	/* Different gateway has different sample layout requirement
+	 * (1) hda host gateway: 24 bit sample will be in lsb bits
```

Comment:
maybe just use capital HDA or HD audio codec will resolve the CI codestyle issue.

Reply:
this is fine.